src/Credentials/ImpersonatedServiceAccountCredentials.php

```diff
@@ -71,10 +77,12 @@ class ImpersonatedServiceAccountCredentials extends CredentialsLoader implements
      *     @type int                            $lifetime The lifetime of the impersonated credentials
      *     @type string[]                       $delegates The delegates to impersonate
      * }
+     * @param string|null $targetAudience The audience to request an ID token.
      */
     public function __construct(
         $scope,
-        $jsonKey
+        $jsonKey,
```

Comment:
I see there are type annotations on the doc comment, should we type these? Just a thought though.

Reply:
Good catch! added in b49eeb9